pymongo/response.py

```diff
@@ -105,3 +108,9 @@ def socket_info(self):
     def pool(self):
         """The Pool from which the SocketInfo came."""
         return self._pool
+
+    @property
+    def exhaust_ready(self):
```

Comment:
I think this would be better named as `more_to_come` because that's what the field tracks.

Reply:
Done.